extra_scripts/create_ucf101_data_files.py

```diff
@@ -0,0 +1,122 @@
+import argparse
```

Comment:
should we document this under https://github.com/facebookresearch/vissl/tree/master/vissl/data#vissl-datasets ? :)

Reply:
Should be fixed in d4e79d89ab08c9caf4f30a0a9157bcc12983b4be.